pkgs/development/php-packages/composer/default.nix

```diff
@@ -33,27 +33,17 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     owner = "composer";
     repo = "composer";
     tag = finalAttrs.version;
-    hash = "sha256-m4CfWWbrmMN0j27XaMx/KRbFjpW5iMMNUlAtzlrorJc=";
+    hash = "sha256-/E/fXh+jefPwzsADpmGyrJ+xqW5CSPNok0DVLD1KZDY=";
   };
 
-  patches = [
-    # Fix an issue preventing reproducible builds
-    # This patch should be removed at the next release (2.8.5)
-    # More information at https://github.com/composer/composer/pull/12090
-    (fetchpatch {
-      url = "https://github.com/composer/composer/commit/7b1e983ce9a0b30a6369cda11a7d61cca9c1ce46.patch";
-      hash = "sha256-veBdfZxzgL/R3P87GpvxQc+es3AdpaKSzCX0DCzH63U=";
-    })
-  ];
-
   nativeBuildInputs = [ makeBinaryWrapper ];
 
   buildInputs = [ php ];
 
   vendor = stdenvNoCC.mkDerivation {
     pname = "${finalAttrs.pname}-vendor";
 
-    inherit (finalAttrs) src version patches;
+    inherit (finalAttrs) src version;
```

Comment:
Could we leave patches so we don't forget about it in future?

Reply:
Argh, saw your comment too late, sorry :(